server/src/test/java/org/opensearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -682,8 +682,7 @@ public static NodeStats createNodeStats(boolean remoteStoreStats) throws IOExcep
                 );
             }
             JvmStats.Classes classes = new JvmStats.Classes(randomNonNegativeLong(), randomNonNegativeLong(), randomNonNegativeLong());
-            jvmStats = frequently()
```

Comment:
As per my understanding,  the `frequently()` here introduces the variation of payload (stats) when `jvmStats` metrics are not populated. If we remove that, this particular case won't be covered, right? What is exactly the problem with it?